frontend/src/entities/emr/detail/emr-clusters-detail.tsx

```diff
@@ -65,15 +67,15 @@ function EMRDetail () {
     }, [dispatch, basePath, projectName, clusterId, clusterName]);
 
     // Refresh data in the background to keep state fresh
-    const isBackgroundRefreshing = useBackgroundRefresh(() => {
-        dispatch(getEMRCluster(clusterId!));
+    const isBackgroundRefreshing = useBackgroundRefresh(async () => {
+        await dispatch(getEMRCluster(clusterId!));
     }, [dispatch]);
 
     const clusterSummary = new Map<string, ReactNode>();
 
     clusterSummary.set('Cluster ID', cluster?.Id);
     clusterSummary.set('Master DNS name', cluster?.MasterPublicDnsName);
-    clusterSummary.set('State', prettyStatus(cluster?.Status?.State));
+    clusterSummary.set('State', prettyStatus(isBackgroundRefreshing ? 'loading' : cluster?.Status?.State));
```

Comment:
This should probably be uppercase.

Reply:
which part? I dont think Github is telling me what you are referring to

Reply:
'loading' instead of 'Loading'

Actually it might be prettyStatus that is failing, but in your gif it 'loading' is lowercase